### ISSUE TITLE
Cannot build fauxton master from CouchDB; CI broken

### ISSUE THREAD

**Issue by wohali** [apache/couchdb-fauxton#1079]
## Expected Behaviour
When updating CouchDB's `rebar.config.script` to point to the latest fauxton (`master` tag instead of `v1.1.14`), Fauxton should build successfully.

## Current Behaviour
```bash
$ make share/www
Building Fauxton
- lodash@4.17.10 node_modules/redux/node_modules/lodash
fauxton@1.1.14 /home/joant/couchdb/src/fauxton
└── redux@4.0.0

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.1.2 (node_modules/watchpack/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
Running "clean:release" (clean) task
>> 2 paths cleaned.

Running "get_deps:default" (get_deps) task
Fetching external dependencies
0 local dependencies

Running "gen_load_addons:default" (gen_load_addons) task

Running "gen_initialize:release" (gen_initialize) task

Running "copy:distDepsRequire" (copy) task
Created 12 directories, copied 218 files

Running "shell:webpackrelease" (shell) task

> fauxton@1.1.14 webpack:release /home/joant/couchdb/src/fauxton
> webpack --optimize-minimize --debug --progress --colors --config ./webpack.config.release.js

Hash: 2758ee8f35e88a8c4be2                                                      Version: webpack 2.2.1
Time: 5940ms
                                               Asset     Size  Chunks             Chunk Names
  dashboard.assets/js/vendor.5b440991272a30c19525.js  35.7 kB    0, 2  [emitted]  vendor
  dashboard.assets/js/bundle.0e253f33f5d9e1080c9f.js  1.49 kB    1, 2  [emitted]  bundle
dashboard.assets/js/manifest.61a726748f3a3781021a.js  1.45 kB       2  [emitted]  manifest
                                          index.html  1.84 kB          [emitted]
  [63] ./~/core-js/modules/es6.array.iterator.js 1.12 kB {0} [built]
  [64] ./~/core-js/modules/es6.object.to-string.js 321 bytes {0} [built]
  [65] ./~/core-js/modules/es6.string.iterator.js 531 bytes {0} [built]
  [66] ./app/main.js 1.39 kB {1} [built] [failed] [1 error]
  [67] ./~/core-js/fn/array/index.js 1.15 kB {0} [built]
  [68] ./~/core-js/fn/promise.js 298 bytes {0} [built]
  [69] ./~/core-js/fn/symbol/index.js 240 bytes {0} [built]
  [70] ./~/regenerator-runtime/runtime.js 24.1 kB {0} [built]
 [106] ./~/core-js/modules/es6.array.sort.js 643 bytes {0} [built]
 [112] ./~/core-js/modules/es7.array.includes.js 379 bytes {0} [built]
 [113] ./~/core-js/modules/es7.promise.finally.js 763 bytes {0} [built]
 [115] ./~/core-js/modules/es7.symbol.async-iterator.js 43 bytes {0} [built]
 [116] ./~/core-js/modules/es7.symbol.observable.js 40 bytes {0} [built]
 [117] ./~/core-js/modules/web.dom.iterable.js 1.81 kB {0} [built]
 [118] multi core-js/fn/array core-js/fn/symbol core-js/fn/promise regenerator-runtime/runtime ./app/main.js 76 bytes {1} [built]
    + 104 hidden modules

ERROR in ./app/main.js
Module build failed: ReferenceError: Unknown plugin "transform-async-to-generator" specified in "/home/joant/couchdb/src/fauxton/.babelrc" at 4, attempted to resolve relative to "/home/joant/couchdb/src/fauxton"
    at /home/joant/couchdb/src/fauxton/node_modules/babel-core/lib/transformation/file/options/option-manager.js:180:17
    at Array.map (native)
    at Function.normalisePlugins (/home/joant/couchdb/src/fauxton/node_modules/babel-core/lib/transformation/file/options/option-manager.js:158:20)
    at OptionManager.mergeOptions (/home/joant/couchdb/src/fauxton/node_modules/babel-core/lib/transformation/file/options/option-manager.js:234:36)
    at OptionManager.init (/home/joant/couchdb/src/fauxton/node_modules/babel-core/lib/transformation/file/options/option-manager.js:368:12)
    at File.initOptions (/home/joant/couchdb/src/fauxton/node_modules/babel-core/lib/transformation/file/index.js:212:65)
    at new File (/home/joant/couchdb/src/fauxton/node_modules/babel-core/lib/transformation/file/index.js:135:24)
    at Pipeline.transform (/home/joant/couchdb/src/fauxton/node_modules/babel-core/lib/transformation/pipeline.js:46:16)
    at transpile (/home/joant/couchdb/src/fauxton/node_modules/babel-loader/lib/index.js:46:20)
    at Object.module.exports (/home/joant/couchdb/src/fauxton/node_modules/babel-loader/lib/index.js:163:20)
 @ multi core-js/fn/array core-js/fn/symbol core-js/fn/promise regenerator-runtime/runtime ./app/main.js
Child html-webpack-plugin for "index.html":
       [0] ./~/html-webpack-plugin/~/lodash/lodash.js 540 kB {0} [built]
       [1] (webpack)/buildin/global.js 509 bytes {0} [built]
       [2] (webpack)/buildin/module.js 517 bytes {0} [built]
       [3] ./~/html-webpack-plugin/lib/loader.js!./assets/index.underscore 2.17 kB {0} [built]

npm ERR! Linux 4.9.0-5-amd64
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "run" "webpack:release"
npm ERR! node v6.14.1
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! fauxton@1.1.14 webpack:release: `webpack --optimize-minimize --debug --progress --colors --config ./webpack.config.release.js`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the fauxton@1.1.14 webpack:release script 'webpack --optimize-minimize --debug --progress --colors --config ./webpack.config.release.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the fauxton package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     webpack --optimize-minimize --debug --progress --colors --config ./webpack.config.release.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs fauxton
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls fauxton
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/joant/couchdb/src/fauxton/npm-debug.log
Warning: Command failed: npm run webpack:release
                                                                                npm ERR! Linux 4.9.0-5-amd64
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "run" "webpack:release"
npm ERR! node v6.14.1
npm ERR! npm  v3.10.10
npm ERR! code ELIFECYCLE
npm ERR! fauxton@1.1.14 webpack:release: `webpack --optimize-minimize --debug --progress --colors --config ./webpack.config.release.js`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the fauxton@1.1.14 webpack:release script 'webpack --optimize-minimize --debug --progress --colors --config ./webpack.config.release.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the fauxton package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     webpack --optimize-minimize --debug --progress --colors --config ./webpack.config.release.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs fauxton
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls fauxton
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/joant/couchdb/src/fauxton/npm-debug.log
 Use --force to continue.

Aborted due to warnings.
Makefile:418: recipe for target 'share/www' failed
make: *** [share/www] Error 6


## Steps to Reproduce (for bugs)
```
git checkout https://github.com/apache/couchdb
# edit rebar.config.script to point to the `master` tag for the `fauxton` repository
# ./configure -c
# make fauxton
```

The Makefile, in reality, does this:

```
$ cd src/fauxton && npm install --production && ./node_modules/grunt-cli/bin/grunt couchdb
```

## Context
CouchDB Jenkins CI builds are broken because of renamed npm modules, so the tagged `v1.1.14` doesn't work. Unfortunately, neither does `master`, so we have a completely busted CI pipeline right now :(

## Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->
* Version used: master
* Browser Name and version: n/a
* Operating System and version (desktop or mobile): Linux / debian 9 / node 6.14.1 / npm 3.10.10

**Comment by popojargo** [comment]
This is just a matter of what packages are installed. The missing component is part of the devDependencies.  I can create a PR for that but I don't have the access to merge that

**Comment by wohali** [comment]
We should be able to build fauxton without installing the devDependencies, correct? There's a reason we have `--production` in the CouchDB tree.

**Comment by popojargo** [comment]
PR on its way...

**Comment by wohali** [comment]
Fixed by #1080, and a PR that's about to hit couchdb to bump us to the new v1.1.15 tag.

Thanks!
